www/include/Administration/parameters/DB-Func.php

```diff
@@ -49,6 +49,25 @@ function updateOption($pearDB, $key, $value)
     $pearDB->query("INSERT INTO `options` (`key`, `value`) VALUES ('$key', $value)");
 }
 
+/**
+ * Used to update fields in the informations table
+ *
+ * @param $pearDB : database connection
+ * @param $key : name of the row
+ * @param $value : value of the row
+ */
+function updateInformations($pearDB, string $key = null, string $value = null)
+{
+    $stmt = $pearDB->prepare("DELETE FROM `informations` WHERE `key` = :key");
```

Comment:
Why not use UPDATE request ?

Reply:
because, the credentials could not have been added prior of the update.
As informations table is has two column : key and value
![image](https://user-images.githubusercontent.com/34628915/92608468-b2bdb180-f2b5-11ea-93df-74e893ead477.png)